sdk/examples/builder_sample.rs

```diff
@@ -67,17 +67,15 @@ where
         }
     ))?;
 
-    // sign a c2pa only manifest store by using a null input stream and application/c2pa as the format.
-    let mut null_stream = Cursor::new([]);
-    let mut output = Cursor::new(Vec::new());
-    builder.sign(
+    // sign a c2pa only manifest store by using a null input and output stream, and application/c2pa as the format.
+    let output = builder.sign(
         &Settings::signer()?,
         "application/c2pa",
-        &mut null_stream,
-        &mut output,
+        &mut io::empty(),
+        &mut io::empty(),
```

Comment:
This changes the behavior of output, so make sure you check that we don't actually use it anywhere, even though this aligns with the comments.